pkg/vbmh/vbmh_test.go

```diff
@@ -25,7 +25,7 @@ var _ = Describe("MachineList", func() {
 		nodes := map[string]*Machine{}
 		for n := 0; n < numNodes; n++ {
 			bmh, _ := testutil.CreateBMH(n, "default", "master", 6)
-			nodes[bmh.Name] = NewMachine(*bmh, airshipv1.VMMaster, NotScheduled)
+			nodes[bmh.Name], _ = NewMachine(*bmh, airshipv1.VMMaster, NotScheduled)
```

Comment:
Here we probably want to do `Expect(err).To(BeNil())`

Reply:
done.